packages/ui-a11y-content/tsconfig.build.json

```diff
@@ -1,7 +1,21 @@
 {
   "extends": "../../tsconfig.build.json",
   "compilerOptions": {
-    "outDir": "./types"
+    "outDir": "./types",
+    "composite": true,
+    "rootDir": "./src"
   },
-  "include": ["src/**/*"]
+  "include": ["src"],
+  "references": [
+    { "path": "../console/tsconfig.build.json" },
+    { "path": "../emotion/tsconfig.build.json" },
+    { "path": "../shared-types/tsconfig.build.json" },
+    { "path": "../ui-dom-utils/tsconfig.build.json" },
+    { "path": "../ui-prop-types/tsconfig.build.json" },
+    { "path": "../ui-react-utils/tsconfig.build.json" },
+    { "path": "../ui-utils/tsconfig.build.json" },
+    { "path": "../uid/tsconfig.build.json" },
+    { "path": "../ui-babel-preset/tsconfig.build.json" },
+    { "path": "../ui-test-utils/tsconfig.build.json" }
```

Comment:
we will need to keep this in sync package.json dependencies & devDependencies?

Reply:
Basically yes, when I did this work I wrote a helper script which generated these reference paths from the package's `package.json` dependencies and devDependencies entries.

Reply:
Can you rerun that script with the fresh master? I'm pretty sure I've added some new deps in my PR-s since then, maybe others have too, so would be nice to update these too. Or if you don't have the script anymore, I'll try to find the ones that were added since :D

Also, just a question: what happens, if a dep is missing from here?

Reply:
Sadly I deleted the script, but I agree that keeping these in sync is a pain in the ass to do manually, so maybe it should be checked automatically at some point of our CI process. (maybe before commit/push?)

To answer the second question:
If no other package is referencing that dep, then it will not get built by `tsc` I guess, I haven't tried it out to be honest.

Reply:
I sent you the dep changes I found on slack. Some of those might be added already, didn'T know how far back shall I check. 

Can you please make tickets for trying to do this automatically and for the output coloring options? Thanks!